packages/renderer/src/lib/preferences/PreferencesAuthenticationProvidersRendering.svelte

```diff
@@ -45,48 +50,54 @@ import DropdownMenuItem from '../ui/DropDownMenuItem.svelte';
-                  <Fa
-                    class="h-3 w-3 text-md mr-2 text-{provider.accounts.length > 0 ? 'green' : 'gray'}-500"
-                    icon="{faCircle}" />
-                </dif>
-                <div class="uppercase text-xs text-{provider.accounts.length > 0 ? 'green' : 'gray'}-500">
-                  <span>
-                    {provider.accounts.length > 0 ? 'Logged in' : 'Logged out'}
-                  </span>
+
+            <!-- Authentication Provider name and status item start -->
+            <div class="px-5 py-2 text-sm m-auto">
+              <div class="flex flex-col">
+                <div class="flex items-center text-lg w-full h-full">
+                  {provider.displayName}
+                </div>
+                <div class="flex flex-row items-center w-full h-full">
+                  <dif>
+                    <Fa
+                      class="h-3 w-3 text-md mr-2 text-{provider.accounts.length > 0 ? 'green' : 'gray'}-500"
+                      icon="{faCircle}" />
+                  </dif>
+                  <div class="uppercase text-xs text-{provider.accounts.length > 0 ? 'green' : 'gray'}-500">
+                    <span>
+                      {provider.accounts.length > 0 ? 'Logged in' : 'Logged out'}
+                    </span>
+                  </div>
                 </div>
-                {#if provider.accounts.length > 0}
-                  <button
-                    aria-label="Sign out of {provider.accounts[0].label}"
-                    class="pl-2 hover:cursor-pointer hover:text-white text-white"
-                    on:click="{() =>
-                      window.requestAuthenticationProviderSignOut(provider.id, provider.accounts[0].id)}">
-                    <Fa class="h-3 w-3 text-md mr-2" icon="{faRightFromBracket}" />
-                  </button>
-                {/if}
               </div>
             </div>
-          </div>
-          <!-- Authentication Provider name and status item end -->
 
-          <!-- Authentication Provider Session label start -->
-          <div class="pt-3 pb-3 text-sm w-2/3 m-auto">
-            <div class="flex flex-row">
-              <div class="flex items-center w-full">
-                <span>
-                  {provider.accounts.length > 0 ? provider.accounts[0].label : ''}
-                </span>
+            {#if provider?.accounts?.length > 0}
+              <!-- Authentication Provider Session label start -->
+              <div class="pt-3 pb-3 text-sm">
+                {#each provider.accounts as account}
+                  <div class="flex flex-row">
+                    <div class="flex items-center w-full">
+                      <div class="text-xs bg-charcoal-800 p-2 rounded-lg mt-1">
+                        <span class="my-auto font-bold col-span-1 text-right">
+                          {account.label}
+                        </span>
+                        <Tooltip tip="Sign out of {account.label}" left>
```

Comment:
You can just give the button a title (which is used as a tooltip) instead of using a Tooltip.

Reply:
<img width="281" alt="image" src="https://github.com/containers/podman-desktop/assets/42176370/8a6dd700-6186-4b1c-9baf-fa77dee4bb2f">

The button tooltip is a bit _unstyled_ (ugly), do we want to use that ?

Reply:
We don't really have a standard, and have used button/other tooltip for most simple cases and Tooltip for cases where it isn't supported or needs more styling. I'm fine leaving it, let's merge.